apps/shlink/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.9"
+services:
+  shlink:
+    container_name: shlink
+    image: ghcr.io/shlinkio/shlink:3.7.0
+    environment:
+      - DEFAULT_DOMAIN=${APP_DOMAIN}
+      - IS_HTTPS_ENABLED=true
+      - DB_DRIVER=postgres
+      - DB_HOST=shlink-db
+      - DB_NAME=shlink
+      - DB_USER=shlink
+      - DB_PASSWORD=shlink
```

Comment:
Here the db password should be randomized using an [env variable](https://runtipi.io/docs/contributing/adding-a-new-app#user-defined-environment-variables) with a type `random`

Reply:
[Here](https://github.com/runtipi/runtipi-appstore/blob/a61a33402037e31905d0c86c9fe110828bdbdda8/apps/activepieces/config.json#L22) is an example of how it was done for Activepieces